README.md

```diff
@@ -1,262 +1,9 @@
-## csv2bufr
+Overview
+========
 
-[![Tests](https://github.com/wmo-im/csv2bufr/workflows/tests%20%E2%9A%99%EF%B8%8F/badge.svg)](https://github.com/wmo-im/csv2bufr/actions/workflows/tests.yml)
+The csv2bufr Python module contains both a command line interface and an API to convert data stored in a CSV file to the WMO BUFR data format.
+More information on the BUFR format can be found in the [WMO Manual on Codes, Volume I.2](https://library.wmo.int/doc_num.php?explnum_id=10722).
 
-csv2bufr is a Python package to transform CSV data into WMO BUFR.  Currently, csv2bufr converts each row in a 
-CSV to a BUFR message. Both a Python module and CLI to the module are provided.
+Documentation for csv2bufr can be found at: [wmo-im.github.io/csv2bufr](https://wmo-im.github.io/csv2bufr/)
```

Comment:
Suggest the README have a very short set of steps (without much explanation) to give the user a sense of usage.

Reply:
README updated.